chain/chain/src/doomslug.rs

```diff
@@ -712,7 +718,11 @@ impl Doomslug {
                     DoomslugBlockProductionReadiness::ReadySince(when) => {
                         if has_enough_chunks {
                             if log_block_production_info {
-                                info!("ready to produce block @ {}, has enough approvals for {:?}, has enough chunks", target_height, now.saturating_duration_since(when));
+                                info!(
+                                    target: "doomslug",
+                                    target_height,
+                                    enough_approvals_for = ?now.saturating_duration_since(when),
+                                    "ready to produce block, has enough approvals, has enough chunks");
```

Comment:
armchair observation: whether the amount of approvals and chunks are sufficient could also each be a field in these three events here.

Reply:
Done